Core/clim-basic/extended-streams/stream-input.lisp

```diff
@@ -68,13 +68,16 @@
   (let ((new-char (car (rassoc char +read-char-map+))))
     (or new-char char)))
 
+(defclass input-kernel-mixin ()
```

Comment:
Why not `input-buffer-mixin`? Input kernel doesn't seem to be a defined concept.

Reply:
Input kernel is mentioned i.e here: http://bauhh.dyndns.org:8000/clim-spec/22-1.html#_1056, in next commits I introduce `input-stream-kernel` which is mentioned in the spec in a few places but not directly specified.